recipes-bsp/formfactor/formfactor/generic-armel-iproc/machconfig

```diff
@@ -0,0 +1,3 @@
+# Assume a USB mouse and keyboard are connected
+HAVE_TOUCHSCREEN=n
+HAVE_KEYBOARD=n
```

Comment:
The docs use {0,1} in their example, unless there is some new standard I could not find: https://docs.yoctoproject.org/bsp-guide/bsp.html?highlight=have_keyboard#miscellaneous-bsp-specific-recipe-files

```suggestion
HAVE_TOUCHSCREEN=0
HAVE_KEYBOARD=0
```

Reply:
It really doesn't matter since these are only evaluated in code for the window manager, and do not install one, but updated it anyway.